frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/links/DisplayLinkToModerationRequest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
+ * With contributions by Bosch Software Innovations GmbH, 2016.
```

Comment:
the file was added as a whole in this PR? in case yes, ... what was the contrib from BSI here?